src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -63,6 +63,41 @@ public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBi
             return actionArguments;
         }
 
+        public async Task<ModelBindingResult> BindModelAsync(
```

Comment:
this class has a `<summary/>` but no member-level doc comments.  prefer if you add comments for new `public` method but at least open a "task" issue to do the necessary.